### PR TITLE
Hock-Schittkowski problems

### PR DESCRIPTION
**Pull request by abelsiqueira**

### CODE REVIEW

File: src/hs01.jl [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#   Lecture Notes in Economics and Mathematical Systems 187,
+#   Springer Verlag Berlin Heidelberg, 1981
+#   10.1007/978-3-642-48320-2
+#
+# A. S. Siqueira, Curitiba/BR, 04/2018.
+
+export hs01
+
+"Hock-Schittkowski problem 1 in NLS format"
+function hs01()
```
Comment (dpo) [inline src/hs01.jl]: Isn't this the same as `mgh01`?

File: src/hs25.jl [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#
+# A. S. Siqueira, Curitiba/BR, 04/2018.
+
+export hs25
+
+"Hock-Schittkowski problem 25 in NLS format"
+function hs25(;nequ = 99)
+
+  u = [25 + (-50 * log(0.01i))^(2/3) for i = 1:nequ]
+  F(x) = [-0.01i + exp(-(u[i] - x[2])^x[3] / x[1]) for i = 1:nequ]
```
Comment (dpo) [inline src/hs25.jl]: I don't have the book with me to check. Are there really complex numbers here?

Comment (abelsiqueira) [reply src/hs25.jl]: You mean `0.01i`? The `i` is an index.

Comment (dpo) [reply src/hs25.jl]: Could we write i/100 instead?

Comment (abelsiqueira) [reply src/hs25.jl]: Done.

Comment (dpo) [reply src/hs25.jl]: Thank you!

**Pull request merged**